### ISSUE TITLE
Large empty space

### ISSUE THREAD

**Issue by SunSea88** [musically-ut/lovely-forks#102]
Hello,

Below the notable forks of GitHub projects, a large empty space appears...
But it disappears if your extension is disabled.

Do you have any idea what could be wrong please ?

Thank you in advance.

Chrome extension

**Comment by musically-ut** [comment]
Good catch! I didn't even notice it because I always have the extension turned on. :D 

I will investigate it.

**Comment by NickAcPT** [comment]
I just noticed this too and decided to check if there were any open issues about said empty space.

I believe it has to do with this CSS style here.
https://github.com/musically-ut/lovely-forks/blob/a2536619d5f6f1c49d69a405fd5cbc71bd5cc86e/webext/data/style.css#L2-L4

**Comment by musically-ut** [comment]
I think that indeed might be it!

I'd spent a few minutes looking into it and had noticed that the block below had been "collapsed" by GitHub by using `margin-top: -16px`. I think moving to `margin-bottom: 20px` might fix or ameliorate the issue. In the worst case, I'll make the style conditional via Javascript.

I'll test it later today. Thank you for looking into it!

**Comment by musically-ut** [comment]
This is now reviewed and published on the webstore!

**Comment by SunSea88** [comment]
Thank you very much ☺️
